packages/components/molecules/f-tabs/src/components/Tab.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <transition
+        v-if="animateTab"
+        :name="transitionName">
+        <div
+            v-show="isActive"
+            :class="[$style['c-tab']]">
+            <slot />
+        </div>
+    </transition>
+    <div v-else>
+        <div
+            v-show="isActive"
+            :class="[$style['c-tab']]">
+            <slot />
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'Tab',
+    props: {
+        title: {
+            required: true,
+            type: String
+        },
+        name: {
+            required: true,
+            type: String
+        },
+        selected: {
+            type: Boolean,
+            default: false
+        }
+    },
+    inject: ['register', 'tabsComponent'],
+    computed: {
+        isActive () {
+            return this.tabsComponent.activeTab === this.name;
+        },
+        transitionName () {
+            return this.tabsComponent.animationDirection === 'LEFT' ?
+                this.$style['fade-in-right'] : this.$style['fade-in-left'];
+        },
+        animateTab () {
+            return this.tabsComponent.animate;
+        }
+    },
+    created () {
+        this.register({
+            name: this.name,
+            title: this.title,
+            selected: this.selected
+        });
+    }
+};
+</script>
+
+<style lang="scss" module>
+/* stylelint-disable -- does not understand syntax. */
```

Comment:
Out of interest – which syntax doesn't it understand? (is it the keyframes?)

Reply:
It's actually the `:global(-enter-to) {` that causes the issue. For some reason it does not see it as valid syntax. I will reduce the disabled area scope to just those bits though